.clang-format

```diff
@@ -0,0 +1,16 @@
+BasedOnStyle:  Google
```

Comment:
Is this necessary? Not a blocker, but also seems irrelevant for most framework users / authors.

Reply:
Yes it's basically the style used for all options not specifically set in the configuration: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#configurable-format-style-options